src/js/photoswipe.js

```diff
@@ -123,6 +330,7 @@ class PhotoSwipe extends PhotoSwipeBase {
     this.offset.y = window.pageYOffset;
 
     this._initialItemData = this.getItemData(this.currIndex);
+    // @ts-expect-error Expected 1-2 arguments, but got 4 ¯\_(ツ)_/¯
     this.dispatch('gettingData', this.currIndex, this._initialItemData, true);
```

Comment:
This should be fixed maybe?

Reply:
yea, it's a typo from a legacy event, will fix it myself. thanks!